lib/helper/Playwright.js

```diff
@@ -234,6 +234,7 @@ class Playwright extends Helper {
       keepBrowserState: false,
       show: false,
       defaultPopupAction: 'accept',
+      ignoreHTTPSErrors: false, //Adding it here o that context can be set up to ignore the SSL errors
```

Comment:
Maybe we should ignore https errors by default? I know most of people run their tests on staging / dev environments where certificates can be invalid...

What do you think?

Reply:
Some of the companies have there sanity as well in production so having this false by default makes sense for that scenario but giving them the ability to override this in lower environments makes sense to me

Reply:
@DavertMik Any idea when it will be merged. My organization relies on this

Reply:
Ah, ok this is default. So sure this is ok!